src/test/java/org/carlspring/cloud/storage/s3fs/spike/ProviderSpecTest.java

```diff
@@ -51,31 +51,6 @@ public void readNothing()
         assertTrue(Files.notExists(base.resolve("file1.html")));
     }
 
-    // FIXME @Test
-    public void seekable()
-            throws IOException
```

Comment:
We might need this test despite it being marked as `FIXME`. There is a difference between `StandardOpenOption.CREATE, StandardOpenOption.WRITE, StandardOpenOption.READ` in `seekable()` and  `StandardOpenOption.APPEND` in `seekableRead()`.